packages/merge/src/components/FieldInput/inputProvider.spec.js

```diff
@@ -0,0 +1,33 @@
+import inputProvider from './inputProvider'
+import {shallow, mount} from 'enzyme'
+import StringInput from './typeInputs/StringInput'
+
+describe('merge', () => {
+  describe('FieldInput Component', () => {
```

Comment:
IMO it's a good idea to group all component tests in a `components` module, so that they appear in the same group in the log (and so that it matches the directory structure; like it's done in the login package)